examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -360,6 +362,71 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             break;
 
           case 'pointmap':
+
+            function byPointsTimeRequest(parameters) {
+              return {
+                dataset: parameters.dataset,
+                filter: getFilter(parameters, defaultPointmapSamplingDayRange, parameters.geoIds),
```

Comment:
I remember you add the time bar chart back today, if that is the case, we should use the time range from the `parameters` other than using the `defaultPointmapSamplingDayRange`.